docs/source/reports/bold.rst

```diff
@@ -87,6 +92,15 @@ reported here.
 
 .. topic:: References
 
+  .. [Patriat2015] Patriat, R., EK Molloy, RM Birn, T. Guitchev, and A. Popov. “Using Edge Voxel Information to 
+     Improve Motion Regression for Rs-FMRI Connectivity Studies.” Brain Connectivity 5, no. 9 (28 2015): 582–95. 
+     doi: `10.1089/brain.2014.0321 <https://doi.org/10.1089/brain.2014.0321>`_.
```

Comment:
```suggestion
     doi: `10.1089/brain.2014.0321 <https://doi.org/10.1089/brain.2014.0321>`__.
```

Reply:
Two underscores is the right choice here.

Reply:
@oesteban what do the one underscore or two underscores do ?

Reply:
LMGTFY...

https://stackoverflow.com/questions/27420317/restructured-text-rst-http-links-underscore-vs-use